source/develop/release-management/features/engine/avoid-ip-spoofing.html.md

```diff
@@ -23,8 +23,8 @@ We should limit traffic that originates from an non-trusted guests to a closed s
 
 ## Current status
 
-*   Link to feature page in a specific release. That release may complete the feature, or parts of it. The complete scope of this feature in this release will be described in the release feature page
-*   Last updated on -- by (WIKI)
+*   Not implemented.
+*   There is workaround with [Network Filter Parameters](http://www.ovirt.org/develop/release-management/features/network/networkfilterparameters/). By setting Network Filter Parameters name as `CTRL_IP_LEARNING` and value as `dhcp`, when combined with vNIC profile's Netowrk Filter set to `clean traffic`, the IP spoofing can be prevented. For avoiding DHCP spoofing the user can also supply parameter with name `DHCPSERVER` and as value use IP address of any trusted DHCP server. The parameters can be provided with IP address manully. By defining multiple (single) parameters with name `IP` and providing IP address (valid on interface) as value.
```

Comment:
Reading this still feels unstructured to me. I would prefer a structure like this:
- `clean-traffic` filter in vNIC Profile prevents IP spoofing
- Network Filter Parameters can be used to restrict the valid IP addresses of a vNIC to a closed set.
- maybe already optional details:
  - Parameter `IP` to list the allowed IP addresses explicitly (the information how to define multiple IP addresses is not required here if you like you can reference https://libvirt.org/formatnwfilter.html#nwfelemsReservedVars for details)
  -  `CTRL_IP_LEARNING` to only honor DHCP server-assigned addresses
  -   `DHCPSERVER` to avoiding DHCP spoofing